tower/src/retry/budget/tps_budget.rs

```diff
@@ -127,7 +125,7 @@ impl Budget {
     }
 }
 
-impl<B: Bucket> BudgetTrait for Budget<B> {
+impl Budget for TpsBudget {
```

Comment:
Just to check we should remove the deposit/withdraw methods from the base impl TpsBudget and only have the trait impl ones. I can't see it in the diff but just want to check.

Reply:
Yes, I already removed them from the base impl.